bin/node/runtime/src/lib.rs

```diff
@@ -1639,6 +1639,16 @@ impl pallet_custom_account::Config<pallet_custom_account::Instance1> for Runtime
 	type WeightInfo = ();
 }
 
+impl pallet_contracts_registry::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type MaxContractContentLen = ConstU32<{2u32 * 1024u32 * 1024u32}>;
+	type MaxSignatures = ConstU32<16u32>;
+	type MaxJudges = ConstU32<16u32>;
+	type AddJudgeOrigin = EnsureRoot<AccountId>;
+	type SubmitOrigin = EnsureSigned<AccountId>;
+	type WeightInfo = ();
```

Comment:
`type WeightInfo = ();` is supposedly discouraged and outdated. Instead, be explicit, for example `	type WeightInfo = pallet_staking::weights::SubstrateWeight<Runtime>;`